### PR TITLE
Allow Miki Mokrysz to review Analytics docs PRs

### PR DESCRIPTION
**Pull request by 46bit**
I'm an engineer on the Data team, particularly focused on Analytics Engine [1]. It makes sense for me to be able to review Analytics PRs.

[1] https://developers.cloudflare.com/analytics/analytics-engine/

### CODE REVIEW

**Comment by pedrosousa** [comment]
I rebuilt the GitHub actions with the updated reviewers list.

**Review by pedrosousa — APPROVED** [review]

File: .github/actions/owners.ts [hunk 1]
```diff
@@ -52,7 +52,6 @@ export const OWNERS: Record<string, string[]> = {
   stream: ["dcpena"],
   tenant: ["kodster28"],
   terraform: ["ranbel"],
-  turnstile: ["patriciasantaana"],
```
Comment (KianNH) [inline .github/actions/owners.ts:55]: Looks like this PR is going to remove Patricia from Turnstile?

Comment (pedrosousa) [reply .github/actions/owners.ts:55]: It was duplicated (two lines below). I got a warning about it when rebuilding the actions.

Comment (KianNH) [reply .github/actions/owners.ts:55]: > It was duplicated (two lines below)

That is a very good point 😅

**Pull request merged**